pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -16,19 +16,19 @@ callPackage ./generic.nix args {
   # check the release notes for compatible kernels
   kernelCompatible = kernel.kernelOlder "6.9";
 
-  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_6;
+  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_8;
 
   # this package should point to a version / git revision compatible with the latest kernel release
   # IMPORTANT: Always use a tagged release candidate or commits from the
   # zfs-<version>-staging branch, because this is tested by the OpenZFS
   # maintainers.
-  version = "2.2.3-unstable-2024-04-09";
-  rev = "28520cad2500b60ce8653e431990e33f77ff08f7";
+  version = "2.2.4";
```

Comment:
Since https://github.com/NixOS/nixpkgs/commit/1f32eb724ddc9e27573266756c390a6bdcca4439#diff-fbbd794729847fb904a20232b805e7dd5781e14f8b66654d6afb59220a649cbcR214 we no longer can check enableUnstable if zfs unstable is used. Before this PR I've used ``(!config.boot.zfs.enabled || config.boot.zfs.package == pkgs.zfs_unstable)`` to check if linuxPackages_latest can be used, but now I need to add another condition if pkgs.zfs and pkgs.zfs_unstable are equal. IMO it was mistake to drop enableUnstable and require people to do Kaffeesatzläserei.

Reply:
Maybe we can add a passthru option to indicate if it's the stable or unstable version.